source/common/config/grpc_mux_impl.cc

```diff
@@ -31,6 +32,26 @@ class AllMuxesState {
   absl::flat_hash_set<GrpcMuxImpl*> muxes_;
 };
 using AllMuxes = ThreadSafeSingleton<AllMuxesState>;
+
+// Returns true if `resource_name` contains the wildcard xdstp resource, for example:
+// xdstp://test/envoy.config.cluster.v3.Cluster/foo-cluster/*
+// xdstp://test/envoy.config.cluster.v3.Cluster/foo-cluster/*?node.name=my_node
+bool isXdstpWildcard(const std::string& resource_name) {
```

Comment:
Nit: XdsTp? Seems other place (hasXdsTpScheme) is using this instead of Xdstp.

Reply:
done